lib/jsonld.js

```diff
@@ -613,34 +605,10 @@ jsonld.fromRDF = util.callbackify(async function(dataset, options) {
     rdfParser = () => dataset;
   }
 
-  // TODO: call `normalizeAsyncFn` on parser fn
-
-  // rdfParser can be callback, promise-based, or synchronous
-  let parsedDataset;
-  if(rdfParser.length > 1) {
-    // convert callback-based rdf parser to promise-based
-    parsedDataset = new Promise((resolve, reject) => {
-      rdfParser(dataset, (err, dataset) => {
-        if(err) {
-          reject(err);
-        } else {
-          resolve(dataset);
-        }
-      });
-    });
-  } else {
-    parsedDataset = Promise.resolve(rdfParser(dataset));
-  }
-
-  parsedDataset = await parsedDataset;
-
-  // back-compat with old parsers that produced legacy dataset format
```

Comment:
Is this removed on purpose, and if so, should it have a changelog comment?

Reply:
I added a note about it.